.travis.yml

```diff
@@ -10,6 +10,10 @@ go:
   - 1.7.x
   - tip
 
+before_install:
+ - sudo add-apt-repository ppa:jacob/virtualisation -y
```

Comment:
Could you share which version we actually require? Do we need to file a bug report against libvirt-go? Because the github repo states that >1.2.0 should be good enough.